src/shell-integration/bash/ghostty.bash

```diff
@@ -6,6 +6,26 @@
 # my simple bash usage this is working. If a bash expert wants to
 # improve this please do!
 
+sudo_function() {
```

Comment:
I don't think we want to leave this function in the global namespace. That was an advantage of the previous approach, along with only defining it when the `sudo` shell feature is enabled.

If we can't find another approach, we should rename this with a `__ghostty_` prefix like our other global functions.